docs/cadt_rpc_api.md

```diff
@@ -1628,64 +1983,342 @@ Functionality: Show the complete history of an organization
+|      Key       |      Type       |                                            Description                                            |
+|:--------------:|:---------------:|:-------------------------------------------------------------------------------------------------:|
+| None (default) |       N/A       | Generate and download a datalayer offer file for a transfer in staging or cancel the active offer |
+|      file      | File Form Field |         A datalayer offer file detailing a transfer of date out of the uploading registry         |
+
+
+
+### GET Examples
+
+#### Generate and download a datalayer offer file
+
+- Used by the offer maker
+- For additional information please see [A Note On CADT Offers](#a-note-on-cadt-offers)
+
+Request
+```shell
+curl --location --request GET 'localhost:31310/v1/offer/' --header 'Content-Type: application/json' > project-offer.txt
+```
+
+Response
+
+Download stream to download the project transfer offer file.
+Using the above `curl` will save the results to a file in the current directory called `project-offer.txt`.
+
+
+#### Get the details of the currently uploaded offer file
+
+- Only fetches purposed changes, makes no changes
+- Used by the offer taker
+- For additional information please see [A Note On CADT Offers](#a-note-on-cadt-offers)
+
+Request
+```shell
+curl --location --request GET 'localhost:31310/v1/offer/accept' --header 'Content-Type: application/json' > project-offer.txt
+```
+
+Response
+
+```json
+TODO
+```
+
+
+### POST Examples
+
+#### Upload an offer file
+
+- Upload and parse an offer file named `offer-file.txt` in the current directory
+- Used by the offer maker
+- For additional information please see [A Note On CADT Offers](#a-note-on-cadt-offers)
+
+```shell
+curl --location --request POST 'localhost:31310/accept/import' --header 'Content-Type: multipart/form-data' \
+--form 'file=./offer-file.txt'
+```
+
+Response
+
+```json
+TODO
+```
+
+### DELETE Examples
+
+#### Cancel the currently active offer
+
+- Used by the offer maker
+- For additional information please see [A Note On CADT Offers](#a-note-on-cadt-offers)
+
+Request
+```shell
+curl --location --request DELETE 'localhost:31310/v1/offer/accept/cancel' --header 'Content-Type: application/json'
+```
+
+Response
+
+```json
+{
+  "message": "Active offer has been canceled",
+  "success": true
+}
+```
+
+#### Reject the currently imported transfer offer file
+
+- Used by the offer taker
+- For additional information please see [A Note On CADT Offers](#a-note-on-cadt-offers)
+
+Request
+```shell
+curl --location --request DELETE 'localhost:31310/v1/offer/accept/cancel' --header 'Content-Type: application/json'
+```
+
+Response
+
+```json
+{
+  "message": "Offer Cancelled",
+  "success": true
+}
+```
+
+### Additional offer resources
+
+* POST `offer/accept/commit` - (offer maker) accept and commit the currently uploaded offer file
+
+
+## `governance`
+
+Functionality: create a governance node and manage governance data
```

Comment:
Change this to "Functionality: For most use cases, read governance values. For development use cases, create and manage governance data."

Reply:
noted